src/core/loader/3dsx.cpp

```diff
@@ -221,8 +253,36 @@ static THREEDSX_Error Load3DSXFile(Core::System& system, FileUtil::IOFile& file,
         }
     }
 
+    // Detect and fill _prm structure
+    PrmStruct pst;
+    std::memcpy(&pst, program_image.data() + sizeof(u32), sizeof(PrmStruct));
+    if (pst.magic == MakeMagic('_', 'p', 'r', 'm')) {
+        static constexpr u32 Argc = 1;
+        static constexpr std::string_view Argv = "Citra";
```

Comment:
```suggestion
        static constexpr u32 argc = 1;
        static constexpr std::string_view argv = "Citra";
```
There seem to be many places inconsistent naming of constexpr's